assets/scss/comments-and-meta.scss

```diff
@@ -190,9 +190,9 @@
           margin-left: 10px;
           border-radius: 4px;
           border: 2px solid;
-          background: #fff;
-          color: #005eb8;
-          border-color: #005eb8;
+          background: $color_nhsuk-white;
```

Comment:
Should this be background-color is only specifying the color and not a full background?